Sources/Subs-BoardIndex.php

```diff
@@ -470,13 +544,141 @@ function getBoardIndex($boardIndexOptions)
 		$context['latest_post'] = $latest_post['ref'];
 	}
 
+	// I took my time, I hurried up, the choice was mine I didn't think enough
+	$parsed_descriptions = setparsedDescriptions($to_parse);
```

Comment:
Just a tiny nitpick here... sorry... case doesn't match

Reply:
case?

Reply:
i guess it should be setParsedDescriptions